pkg/visitors/assemble.go

```diff
@@ -242,6 +242,35 @@ func (v *Assemble) Visit(f uefi.Firmware) error {
 				binary.LittleEndian.PutUint16(newBuf, f.BuildNumber)
 				newBuf = append(newBuf, unicode.UTF8ToUCS2(f.Version)...)
 				f.SetBuf(newBuf)
+			case uefi.SectionTypeDXEDepEx, uefi.SectionTypePEIDepEx,
+				uefi.SectionMMDepEx:
```

Comment:
There are a few more sanity checks you could do, but I'm not sure if they are worth it. For example, PEIDepEx does not support all the opcodes. Also the PI spec says that "NoteThe PEI Foundation will only support the evaluation of dependency expressions that are less than or equal to 256 terms"

Reply:
I'll add a TODO